ctl/ctl.go

```diff
@@ -60,9 +60,12 @@ func (a *API) Get(get Get, reply *GetReply) (err error) {
 	return
 }
 
-func (a *API) Forget(forget Forget) {
+func (a *API) Forget(forget Forget, ok *bool) error {
 	log.Info().Msgf("Forget: %s", forget.Key)
 	a.dht.Forget(forget.Key)
+
+	*ok = true
+	return nil
```

Comment:
Nu är jag kanske jäkligt trött... men kommer denna funktionen någonsin kunna returnera ett error?

Reply:
Näh, men den uppfyller RPC interfacet :)

Reply:
Precis :grin: 
Jag tänker så här dock: `dht.Forget()` kan faktiskt misslyckas med att ta bort ett värde av olika anledningar... och ingen av funktionerna neråt i hierarkin varken tar hand om eller returnerar errors :thinking: Kanske borde hanteras i store?

Reply:
Den använder `delete` på en map, som inte returnerar något error. I så fall måste man först göra en lookup och sen göra en delete

Reply:
Jag kan vara lite med på det tåget dock. Just nu sitter jag och debuggar varför inte värden försvinner.

Reply:
> Den använder `delete` på en map, som inte returnerar något error. I så fall måste man först göra en lookup och sen göra en delete

Hm... så pass. Det låter dock rätt att göra lookup först, sen delete.

> Jag kan vara lite med på det tåget dock. Just nu sitter jag och debuggar varför inte värden försvinner.

Så `Forget` funkar inte riktigt? 

Ska vi dra in denna i master och sen göra en issue på det... eller hur gör vi?

Reply:
Jag fixar nog detta nu när jag panik refactorar